themes/default/content/blog/stack-readme/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Stack READMEs in the Pulumi Service"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-05-16T14:35:01-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: Starting today you can add a README to your Pulumi Service Stacks to store key links, CLI commands and documentation.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - evan-boyle
+    - devon-grove
+    - myles-haynes
+    - casey-huang
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
```

Comment:
Minor - but I tend to like having a sentence or paragraph that sets up the opportunity.

In this case, I think it's along the lines of "Each Pulumi Stack you deploy manages a key set of cloud infrastructure for your organization.  The Pulumi Console includes a variety of features for exposing key information about your stack for all other users within your organization - configuration, outputs, resources under management, deep links to cloud providers, and a graph of all resources.  However, it's often useful to allow the author of a Pulumi stack to describe in their own words the key elements of a stack so that future viewers can quickly understand the components and cloud resources that are managed."  (Could be worded much better :-) - but something along those lines).

Reply:
I do see you you have something along these lines a few paragraphs in.  Perhaps okay to leave that as is.  Or include both.  Up to you!